docs/sources/administration/configuration.md

```diff
@@ -1201,18 +1201,22 @@ Configure Grafana's Jaeger client for distributed tracing.
 You can also use the standard `JAEGER_*` environment variables to configure
 Jaeger. See the table at the end of https://www.jaegertracing.io/docs/1.16/client-features/
 for the full list. Environment variables will override any settings provided here.
+These environment variables follow the `GF_<section>` convention above,
+i.e. `JAEGER_SAMPLER_TYPE` becomes `GF_TRACING_JAEGER_SAMPLER_TYPE`.
 
 ### address
 
 The host:port destination for reporting spans. (ex: `localhost:6831`)
 
-Can be set with the environment variables `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT`.
+Can be set with the environment variables `GF_TRACING_JAEGER_AGENT_HOST`
```

Comment:
I have actually submitted an issue relating to these env vars as they do not enable tracing - `address` must be set, unlike it states here. https://github.com/grafana/grafana/issues/32667

Reply:
This doesn't look right. You can actually set it either using `GF_TRACING_JAEGER_ADDRESS` or either using `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT`, see https://www.jaegertracing.io/docs/1.16/client-features/.

Reply:
I used `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT` initially and it did not initialize tracing for me. Let's look at `tracing.go`:

```go
func (ts *TracingService) Init() error {
	ts.log = log.New("tracing")
	ts.parseSettings()

	if ts.enabled {
		return ts.initGlobalTracer()
	}

	return nil
}
```

`ts.initGlobalTracer()` is where the Jaeger Cfg is parsed:

```golang
func (ts *TracingService) initGlobalTracer() error {
	cfg, err := ts.initJaegerCfg()
	if err != nil {
		return err
	}
....
```

And it'll only go to `ts.initGlobalTracer()` if `ts.enabled`. So let's return to `ts.ParseSettings()` just before it:

```golang
func (ts *TracingService) parseSettings() {
	var section, err = ts.Cfg.Raw.GetSection("tracing.jaeger")
	if err != nil {
		return
	}

	ts.address = section.Key("address").MustString("")
	if ts.address != "" {
		ts.enabled = true
	}

	ts.customTags = splitTagSettings(section.Key("always_included_tag").MustString(""))
	ts.samplerType = section.Key("sampler_type").MustString("")
	ts.samplerParam = section.Key("sampler_param").MustFloat64(1)
	ts.zipkinPropagation = section.Key("zipkin_propagation").MustBool(false)
	ts.disableSharedZipkinSpans = section.Key("disable_shared_zipkin_spans").MustBool(false)
	ts.samplingServerURL = section.Key("sampling_server_url").MustString("")
}
```

We can see `ts.enabled` is explictly dependent on `GF_TRACING_ADDRESS`.

Maybe the Jaeger config would start up successfully using `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT` (rather than `GF_TRACING_JAEGER_AGENT_PORT`, so the update to the text here isn't relevant, but the issue #32667 is relevant.

Reply:
#32667 is definitely relevant for enable tracing and should be fixed. I thought that actually was fixed, but I was wrong. I'm pretty sure there was some other PR that changed things. Had #23887 in mind, but no it didn't touch upon this problem. #25768 is related, but didn't really do anything in regards to enable tracing. Maybe I started some work on this back in the day, but never ended up in a PR?

However, if you would provide a value for `GF_TRACING_JAEGER_ADDRESS` and values for `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT` I think the latter ones would be used.

Reply:
> However, if you would provide a value for GF_TRACING_JAEGER_ADDRESS and values for JAEGER_AGENT_HOST and JAEGER_AGENT_PORT I think the latter ones would be used.

Yeah definitely.

I think this PR arose from my troubles at #32667 and I think this PR can be closed - after the above comment where I dug a bit further, I realized that these docs would be fine and accurate if #32667 did not exist.